tests/Unit/Domain/DomainCreators/Test_Disk.cpp

```diff
@@ -0,0 +1,204 @@
+    const std::vector<std::array<size_t, 2>>& expected_refinement_level,
+    const bool with_equiangular_map) {
+  const auto domain = disk.create_domain();
+  const auto& blocks = domain.blocks();
+  const OrientationMap<2> aligned_orientation{};
+  const OrientationMap<2> quarter_turn_ccw(std::array<Direction<2>, 2>{
+      {Direction<2>::lower_eta(), Direction<2>::upper_xi()}});
+  const OrientationMap<2> half_turn(std::array<Direction<2>, 2>{
+      {Direction<2>::lower_xi(), Direction<2>::lower_eta()}});
+  const OrientationMap<2> quarter_turn_cw(std::array<Direction<2>, 2>{
+      {Direction<2>::upper_eta(), Direction<2>::lower_xi()}});
+  const std::vector<std::unordered_map<Direction<2>, BlockNeighbor<2>>>
+      expected_block_neighbors{
+          {{Direction<2>::lower_eta(), {3, aligned_orientation}},
+           {Direction<2>::upper_eta(), {1, aligned_orientation}},
+           {Direction<2>::lower_xi(), {4, aligned_orientation}}},
+          {{Direction<2>::lower_eta(), {0, aligned_orientation}},
+           {Direction<2>::upper_eta(), {2, aligned_orientation}},
+           {Direction<2>::lower_xi(), {4, quarter_turn_cw}}},
+          {{Direction<2>::lower_eta(), {1, aligned_orientation}},
+           {Direction<2>::upper_eta(), {3, aligned_orientation}},
+           {Direction<2>::lower_xi(), {4, half_turn}}},
+          {{Direction<2>::lower_eta(), {2, aligned_orientation}},
+           {Direction<2>::upper_eta(), {0, aligned_orientation}},
+           {Direction<2>::lower_xi(), {4, quarter_turn_ccw}}},
+          {{Direction<2>::upper_xi(), {0, aligned_orientation}},
+           {Direction<2>::upper_eta(), {1, quarter_turn_ccw}},
+           {Direction<2>::lower_xi(), {2, half_turn}},
+           {Direction<2>::lower_eta(), {3, quarter_turn_cw}}}};
+  const std::vector<std::unordered_set<Direction<2>>>
+      expected_external_boundaries{{{Direction<2>::upper_xi()}},
+                                   {{Direction<2>::upper_xi()}},
+                                   {{Direction<2>::upper_xi()}},
+                                   {{Direction<2>::upper_xi()}},
+                                   {}};
+
+  const std::vector<std::array<size_t, 2>>& expected_extents{
+      expected_wedge_extents,
+      expected_wedge_extents,
+      expected_wedge_extents,
+      expected_wedge_extents,
+      {{expected_wedge_extents[1], expected_wedge_extents[1]}}};
+
+  CHECK(disk.initial_extents() == expected_extents);
+  CHECK(disk.initial_refinement_levels() == expected_refinement_level);
+  using Wedge2DMap = CoordinateMaps::Wedge2D;
+  using Wedge2DType =
+      CoordinateMap<Frame::Logical, Frame::Inertial, Wedge2DMap>;
+  using AffineMap = CoordinateMaps::AffineMap;
+  using AffineMap2D = CoordinateMaps::ProductOf2Maps<AffineMap, AffineMap>;
+  using Equiangular = CoordinateMaps::Equiangular;
+  using Equiangular2D =
+      CoordinateMaps::ProductOf2Maps<Equiangular, Equiangular>;
+
+  PUPable_reg(
```

Comment:
remove the PUPable_reg statements (they are taken care of in test_domain_construction)